docs/zh_cn/juicefs_on_windows.md

```diff
@@ -1,68 +0,0 @@
-# Windows 系统使用 JuiceFS
```

Comment:
Why this one is deleted?

Reply:
because the content of this document  had been merged to the new installation document https://juicefs.com/docs/zh/community/installation and this old file never indexed to sidebar.